src/scope.js

```diff
@@ -632,16 +633,31 @@ export class FunctionScope extends Scope {
     }
 }
 
-export class ForScope extends Scope {
+ class ForScope extends Scope {
     constructor(scopeManager, upperScope, block) {
         super(scopeManager, 'for', upperScope, block, false);
     }
 }
 
-export class ClassScope extends Scope {
+class ClassScope extends Scope {
     constructor(scopeManager, upperScope, block) {
         super(scopeManager, 'class', upperScope, block, false);
     }
 }
 
+module.exports = {
+    "default": Scope,
```

Comment:
Since this isn't using modules anymore, it might be better to use `module.exports.Scope` instead of `module.exports.default`.